### DISCUSSION TITLE
New feature requests for lazy loading

### DISCUSSION THREAD

**Discussion by Shougo** [vim-denops/denops.vim#75] [Ideas]
1. `call denops#plugin#discover()` should not reload existing denops plugins

2. Add `g:denops#ready` like variable to check `DenopsReady` is already called.

They are needed to support lazy loading in plugin managers.

**Comment by lambdalisue** [comment]
I'll think about it 👍

**Reply by lambdalisue** [reply]
https://github.com/vim-denops/denops.vim/pull/76

**Comment by Shougo** [comment]
It is already merged.
I think it can be closed.
